contactlayout.php

```diff
@@ -137,12 +137,25 @@ function contactlayout_civicrm_pageRun(&$page) {
         foreach ($layout['blocks'] as &$row) {
           foreach ($row as &$column) {
             foreach ($column as &$block) {
-              if (!empty($block['profile_id'])) {
-                $relatedContact = ProfileRelatedContact::get($contactID, !empty($block['related_rel']) ? $block['related_rel'] : '');
-                $profileContact = $relatedContact ? $relatedContact : $contactID;
-                $profileBlocks[$block['profile_id']] = CRM_Contactlayout_Page_Inline_ProfileBlock::getProfileBlock($block['profile_id'], $profileContact);
+              if (empty($block['profile_id'])) {
+                continue;
+              }
+
+              $profileContact = $contactID;
+              $block['rel_cid'] = NULL;
+              $block['rel_isMissing'] = FALSE;
```

Comment:
Can we change this to use snake case all through instead of camel and snake case.
i.e `rel_is_missing`